cupy/sorting/count.py

```diff
@@ -1,18 +1,25 @@
 from cupy import core
 
 
-def count_nonzero(x):
+def count_nonzero(x, axis=None):
     """Counts the number of non-zero values in the array.
 
     Args:
         x (cupy.ndarray): The array for which to count non-zeros.
-
+        axis (int or tuple): optionalAxis or tuple of axes along which to count
```

Comment:
Maybe `axis (int or tuple, optional): Axis or tuple of axes along which to count` is correct?